README.md

```diff
@@ -250,6 +255,13 @@ Click to expand any of the code examples below.
 All of these can be run in a Colab notebook:
 [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/langroid/langroid/blob/main/examples/langroid_quick_examples.ipynb)
 
+We also provide a containerized version of `Langroid` repository via this [Docker Image](https://hub.docker.com/r/mohannadcse/langroid). All what you need is just setting up environment variables under `.env`. Please follow these steps to setup the container. 
+```bash
+docker run -it mohannadcse/langroid /bin/bash
```

Comment:
can we create an organization called `langroid` on dockerhub. I can create it

Reply:
Yes, should I? We can use affiliated langroid email

Reply:
if people already have a .env file, there needs to be instructions on how they can easily copy this to the docker env.

Reply:
How about adding a reference to this section that we already have in README https://github.com/langroid/langroid#set-up-environment-variables-api-keys-etc

Reply:
or you mean copy/paste from host to container or just mounting the .env file?

Reply:
not mounting... I thought there was a way to "copy" a file from host to env

Reply:
sure, there is a command to copy file from host to container, this command assumes the user already has upfront the .env file that exists on the host.